classes/class-wc-qd-checkout-vat.php

```diff
@@ -29,51 +29,32 @@ public function update_taxes_on_update_cart( $cart ) {
 		
 		if( is_cart() ) {
 
-			$tax_based_on = is_cart() ? 'base' : get_option( 'woocommerce_tax_based_on' );
-			// Tax location
-			if ( 'base' === $tax_based_on ) {
-				$country  	= WC()->countries->get_base_country();
-				$state 		= WC()->countries->get_base_state();
-				$postcode 	= WC()->countries->get_base_postcode();
-				$city 		= WC()->countries->get_base_city();
-			} elseif ( 'billing' === $tax_based_on ) {
-				$country  	= sanitize_text_field( $_POST['billing_country'] );
-				$state 		= sanitize_text_field( $_POST['billing_state'] );
-				$postcode 	= sanitize_text_field( $_POST['billing_postcode'] );
-				$city 		= sanitize_text_field( $_POST['billing_city'] );
-			} else {
-				$country  	= sanitize_text_field( $_POST['shipping_country'] );
-				$state 		= sanitize_text_field( $_POST['shipping_state'] );
-				$postcode 	= sanitize_text_field( $_POST['shipping_postcode'] );
-				$city 		= sanitize_text_field( $_POST['shipping_city'] );
-			}
-			$tax_id = sanitize_text_field( 'billing' === $tax_based_on ? $_POST['tax_id'] : '' );
-
-			// Check if the customer is VAT exempted
-			if ( empty( $tax_id ) || false === WC_QD_Tax_Id_Field::is_valid( $tax_id, $country ) ) {
-				WC()->customer->set_is_vat_exempt( false );
-			} else {
-				WC()->customer->set_is_vat_exempt( true );
-			}
+			$tax_based_on 	= 'base';
+			$country  		= WC()->countries->get_base_country();
```

Comment:
You can use the base address if `tax_based_on` is `base` or there's no shipping option. 
Otherwise, you must use the shipping option provided by the customer

![CleanShot 2023-03-10 at 18 42 21@2x](https://user-images.githubusercontent.com/431649/224399255-bdd183c2-f27d-488f-9d59-f5a40f307517.png)